graphql-api.cabal

```diff
@@ -35,11 +35,11 @@ library
     , graphql
     , text
   exposed-modules:
-      GraphQL.Definitions
       GraphQL.Input
       GraphQL.Output
       GraphQL.Schema
-      GraphQL.TypeApi
+      GraphQL.TypedApi
+      GraphQL.TypedSchema
```

Comment:
+1 except for `Literals` because it has quite a different connotation (e.g. [1]). Not sure what else to call it though. Aeson has value in the `Types` module but that doesn't seem great either. Brainstorm:

* Raw
* Direct
* Encoding
* EncodedValue
* Values
* Representation
* ValueTree

It's the in-memory intermediate representation before/after serialization. Not sure what the usual name is.

[1]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types